features/step_definitions/api_credentials_steps.rb

```diff
@@ -1,33 +1,33 @@
+# frozen_string_literal: true
+
 # API v1:
-Given /^(buyer "[^"]*") has user key "([^"]*)"$/ do |buyer, key|
-  buyer.bought_cinstance.update_attribute(:user_key, key)
+Given "{buyer} has user key {string}" do |buyer, key|
```

Comment:
This is technically correct and looks cool. On the other hand as a step user I can't immediately understand what to put for buyer. It is more understandable in case step definition is `buyer {buyer} has user key {string}`, I'm not sure if transformation can work in this case, because `{buyer}` would be same RE as `{string}`. Maybe needs to be tried.

Reply:
Actually seems like it should work when I see how `forum` transformation is defined. I like the `forum` transformation as it leaves the static string visible in the step definition.

Reply:
Or it can be a convention the `{something}` transformation means `something {string}` under the hood. I'm not really sure. Step definition looks nice, just I'm worried about step users legibility.

Reply:
I consider the convention easy to understand. Closing this thread.